docs/training/hyperparameter/search-methods/hp-custom.rst

```diff
@@ -0,0 +1,200 @@
+.. _topic-guides_hp-tuning-det_custom:
+
+#######################
+ Custom Search Methods
+#######################
+
++----------------------------------------------------------------+
+| API reference                                                  |
++================================================================+
+| :doc:`/reference/reference-searcher/custom-searcher-reference` |
++----------------------------------------------------------------+
+
+Determined supports defining your own hyperparameter search algorithms and provides search runner
+utilities for executing them.
+
+.. tip::
+
+   Remember that a Determined experiment is a set of trials, each corresponding to a point in the
+   hyperparameter space.
+
+To implement a custom hyperparameter tuning algorithm, subclass
+:class:`~determined.searcher.SearchMethod`, overriding its event handler methods. Note that
+overriding :meth:`~determined.searcher.SearchMethod.save_method_state` and
+:meth:`~determined.searcher.SearchMethod.load_method_state` of
+:class:`~determined.searcher.SearchMethod` is not mandatory. However, it is *recommended* as it
+enables fault tolerance.
+
+To run the custom hyperparameter tuning algorithm, you can use:
+
+-  :class:`~determined.searcher.LocalSearchRunner` that executes on your machine,
+-  :class:`~determined.searcher.RemoteSearchRunner` that runs on a Determined cluster.
+
+Both Search Runners execute the custom hyperparameter tuning algorithm and start a multi-trial
+experiment on a Determined cluster.
+
+The following sections explain the steps to take in order to implement and use a custom
+hyperparameter search algorithm. A detailed example can be found here:
```

Comment:
This is where I decided to put the link to the example. The reader can choose to work through a detailed example _or_ read the schematic explanation.